README.md

```diff
@@ -30,51 +30,67 @@ https://github.com/conda-forge/miniforge/#download
 
 ## Compile LPython
 
-Create a Conda environment using the preexisting environment.yml file:
-
-    conda env create -f environment.yml
-    conda activate lp
-
 Install required packages (Linux - 64 bit):
 
-    sudo apt install binutils-dev zlib1g-dev
+```bash
+sudo apt install binutils-dev zlib1g-dev
+```
 
 Clone LPython
 
-    git clone https://github.com/lcompilers/lpython.git
-    cd lpython
+```bash
+git clone https://github.com/lcompilers/lpython.git
+cd lpython
+```
+
+Create a Conda environment using the preexisting environment.yml file:
+
+```bash
+conda env create -f environment.yml
+conda activate lp
+```
 
 Create autogenerated files (choose the command for your platform):
 
-    ./build0.sh      # macOS/Linux
-    call build0.bat  # Windows
+```bash
+./build0.sh      # macOS/Linux
+call build0.bat  # Windows
+```
 
 Compile LPython:
 
-    cmake -DCMAKE_BUILD_TYPE=Debug -DWITH_LLVM=yes -DWITH_STACKTRACE=yes -DWITH_LFORTRAN_BINARY_MODFILES=no .
-    cmake --build . -j16
+```bash
+cmake -DCMAKE_BUILD_TYPE=Debug -DWITH_LLVM=yes -DWITH_STACKTRACE=yes -DWITH_LFORTRAN_BINARY_MODFILES=no .
+cmake --build . -j$nproc
```

Comment:
I don't have `$nproc` defined. Is the `-j16` confusing? 

We could switch to ninja, which runs in parallel by default.

Reply:
May be `$nproc` doesnot work on Mac, the reason it was confusing to me was that I only had 8 cores in my CPU but it had hardcoded 16 in the readme, but I guess it wouldn't matter much as it will always take 8 cores if that's the maximum count. 

I can revert this change, may be switching to ninja is an option but IMO not something thats needed right now.